io.github.project516.Shuffleboard.yml

```diff
@@ -0,0 +1,69 @@
+app-id: io.github.project516.Shuffleboard
```

Comment:
https://github.com/Project516/shuffleboard doesn't represent the project. (it's empty) this ID is not acceptable.